test/test_autograd.py

```diff
@@ -2467,8 +2491,10 @@ def test_profiler_aggregation_lstm(self):
                 end = time.time()
                 total_time_s += end - start
 
-        print(prof.table(sort_by="self_cpu_time_total", row_limit=10))
-        print(prof.key_averages().table(sort_by="self_cpu_time_total", row_limit=10))
+        print(prof.table(
```

Comment:
actually I am not sure why we were printing to stdout before, it's not useful in the unit test. can you change it, just call `__repr__` on it for the sanity check.

Reply:
@soumith , is there a specific issue with printing to stdout? Normally a python test runner would capture std out. But when you run a test manually output is useful. Is this is not a case for some of the OSS CIs? 
 
No strong feelings about this though :)

Reply:
the default testrunner for python doesn't capture stdout. But the default one in `buck` eats it properly.
Anyways, this is a nit and it wasn't introduced in your diff.